framework/pym/play/application.py

```diff
@@ -218,7 +218,6 @@ def log_path(self):
         return log_path
 
     def check_jpda(self):
-        self.jpda_port = self.readConf('jpda.port')
```

Comment:
I think he might be referencing that in https://github.com/playframework/play1/pull/835#commitcomment-9746326